deepchecks/vision/suites/default_suites.py

```diff
@@ -14,29 +14,38 @@
 It is possible to customize these suites by editing the checks and conditions inside it after the suites' creation.
 """
 from deepchecks.vision.checks import ClassPerformance, TrainTestLabelDrift, MeanAveragePrecisionReport, \
-                                     MeanAverageRecallReport
+    MeanAverageRecallReport, ImagePropertyDrift, ImageDatasetDrift, SimpleModelComparison, ConfusionMatrixReport, \
+    RobustnessReport
 from deepchecks.vision import Suite
 
 
 __all__ = ['train_test_validation', 'model_evaluation', 'full_suite']
 
+from deepchecks.vision.checks.distribution import HeatmapComparison
+
 
 def train_test_validation() -> Suite:
     """Create a suite that is meant to validate correctness of train-test split, including integrity, \
     distribution and leakage checks."""
     return Suite(
         'Train Test Validation Suite',
-        TrainTestLabelDrift()
+        HeatmapComparison(),
+        TrainTestLabelDrift(),
+        ImagePropertyDrift().add_condition_drift_score_not_greater_than(),
+        ImageDatasetDrift()
     )
 
 
 def model_evaluation() -> Suite:
     """Create a suite that is meant to test model performance and overfit."""
     return Suite(
         'Model Evaluation Suite',
-        ClassPerformance().add_condition_test_performance_not_less_than(0.5),
-        MeanAveragePrecisionReport().add_condition_test_average_precision_not_less_than(0.5),
-        MeanAverageRecallReport().add_condition_test_average_recall_not_less_than(0.5),
```

Comment:
why not have a condition per check like in tabular?

Reply:
Currently these conditions are "too conservative" and will raise a lot of failed conditions, and I'm not sure what magic number we define here will work for different classes. I think we'll need to redefine the condition (perhaps have the threshold be dependent on the number of classes) and I prefer to have this hotfix out for now till we do that.

Reply:
Opened #930 for that